css-ui-4/Overview.bs

```diff
@@ -2199,6 +2206,24 @@ Switching appearance: the 'appearance' property</h3>
 
 			Elements other than <a>widgets</a> must be rendered as for ''appearance/none''.
 
+		<dt><dfn>base</dfn>
+		<dd>
+			The effect of ''appearance/base'' depends on the element it is applied to:
+			<dl class=switch>
+				: Replaced elements other than <a>widgets</a>
```

Comment:
I think you should drop the word "Replaced" here, since with that word dropped the `<dl class=switch>` becomes complete.

Reply:
done